e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -96,9 +97,9 @@ public async Task ProvisioningDeviceClient_RegisterAsyncInvalidServiceCertificat
 
         private static async Task TestInvalidServiceCertificate(ProvisioningTransportHandler transport)
         {
-            using var security =
-                new SecurityProviderX509Certificate(Configuration.Provisioning.GetIndividualEnrollmentCertificate());
-            ProvisioningDeviceClient provisioningDeviceClient = ProvisioningDeviceClient.Create(
+            using X509Certificate2 cert = Configuration.Provisioning.GetIndividualEnrollmentCertificate();
+            using var security = new SecurityProviderX509Certificate(cert);
+            var provisioningDeviceClient = ProvisioningDeviceClient.Create(
```

Comment:
Type would make sense anywhere we don't directly use the 'new' keyword.

Reply:
I think what intellisense infers is:
`objectA.Create()` -> returned type is `objectA` so using `var` is ok
`objectA.CreateSomething()` -> returned type is not obvious, so don't use `var`.

I can update it to be more explicit though, sure.